app/controllers/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+
+  def index
+    @users = User.all
+  end
+
+  def show
```

Comment:
I'm confused about your `UsersController` `show` action. So the only `user` page we will always see is the `current_user` ?

Kindly fix this. The `current_user` should be able to visit all other users' pages.

Reply:
> **Odin project 13:** The User Show page contains their Profile information, photo, and Posts.

Our user show controller is programmed according to this, that was the requirement